cli/cmd/install.go

```diff
@@ -18,131 +18,66 @@ package cmd
 
 import (
 	"bufio"
-	"errors"
 	"fmt"
 	"os"
 	"strings"
 
-	"helm.sh/helm/v3/pkg/chart"
-
-	"github.com/keptn/keptn/cli/pkg/platform"
-
 	"github.com/keptn/keptn/cli/pkg/helm"
-
 	"github.com/keptn/keptn/cli/pkg/kube"
-
 	"github.com/keptn/keptn/cli/pkg/logging"
+	"github.com/keptn/keptn/cli/pkg/platform"
 
 	keptnutils "github.com/keptn/kubernetes-utils/pkg"
 	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/strvals"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type installCmdParams struct {
-	installUpgradeParams
-	UseCaseInput             *string
-	UseCase                  usecase
-	EndPointServiceTypeInput *string
-	EndPointServiceType      endpointServiceType
-}
+var installParams installUpgradeParams
+var keptnInstallChart *chart.Chart
+var installValues = getDefaultValues()
 
-var installParams installCmdParams
-var keptnChart *chart.Chart
 
-// installCmd represents the version command
+// installCmd represents the install command
 var installCmd = &cobra.Command{
 	Use:   "install",
 	Short: "Installs Keptn on a Kubernetes cluster",
 	Long: `The Keptn CLI allows installing Keptn on any Kubernetes derivate to which your kube config is pointing to, and on OpenShift.
 
 For more information, please follow the installation guide [Install Keptn](https://keptn.sh/docs/` + keptnReleaseDocsURL + `/operate/install/#install-keptn)
 `,
-	Example: `keptn install                                                        # install on Kubernetes
-
-keptn install --platform=openshift --use-case=continuous-delivery    # install continuous delivery on Openshift
-
-keptn install --platform=kubernetes --endpoint-service-type=NodePort # install on Kubernetes with gateway NodePort
-`,
+	Example: `keptn install --values=./my-values.yaml`,
```

Comment:
@anukul can you please also add an example for setting values directly with `--set`?

Reply:
@anukul any updates on this?